shortfin/python/shortfin_apps/sd/components/service.py

```diff
@@ -212,8 +215,8 @@ class BatcherProcess(sf.Process):
     into batches.
     """
 
-    STROBE_SHORT_DELAY = 0.1
-    STROBE_LONG_DELAY = 0.25
+    STROBE_SHORT_DELAY = 0.5
+    STROBE_LONG_DELAY = 1
```

Comment:
How does this impact user experience?

Reply:
these are the increments by which the batcher checks for incoming requests